api/src/migrations/api/1611329857558-initialDataMetadataEntities.ts

```diff
@@ -0,0 +1,89 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class initialDataMetadataEntities1611329857558 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query(`
+      CREATE TYPE "status" AS ENUM (
+        'created',
+        'running',
+        'done',
+        'failure'
+      );
+
+      CREATE TYPE "tags" AS ENUM (
+        'bioregional',
+        'species'
+      );
+
+      CREATE TABLE "features" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "feature_data_id" uuid NOT NULL,
+        "feature_class_name" varchar,
+        "alias" varchar,
+        "description" varchar,
+        "property_name" varchar,
+        "intersection" uuid[],
+        "tag" tags NOT NULL,
+        "creation_status" status NOT NULL,
+        "created_at" timestamp NOT NULL default now(),
+        "created_by" uuid NOT NULL,
+        "modify_at" timestamp NOT NULL default now()
+      );
+
+      CREATE TABLE "scenarios" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "name" varchar,
+        "project_id" varchar NOT NULL REFERENCES "projects" ("id"),
+        "iso3" varchar(3) NOT NULL,
```

Comment:
I have created a `countries` table with iso2, iso3, name and a JSONB for dictionary of local names (if ever needed), so we can later update this to reference the countries table (or keep this column out of this migration and just add it via my PR) - if keeping it, I would call it `country_id` though, for clarity.